.github/workflows/check-milestone.yaml

```diff
@@ -8,15 +8,20 @@ on:
   pull_request:
     branches:
       - 'main'
-
+    types:
+      - opened
+      - demilestoned
+      - milestoned
+      - edited
+      - synchronize
+  
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
 jobs:
   check-milestone:
     runs-on: ubuntu-latest
-    if: ${{ github.actor != 'dependabot[bot]' }}  # dependabot prs do not have to be in milestone
```

Comment:
Why removing this?

Reply:
Why do we have this ?

Reply:
We don't want this check to be performed for lib bump PR which is created by the dependant bot.

Reply:
Personally i want this for every PR.

Reply:
We don't want to add all those PRs to the milestone manually 🤔

Reply:
At the end of the day all PRs must have a milestone, without exception.

Reply:
> We don't want to add all those PRs to the milestone manually 🤔

It's not a huge effort to assign a milestone to a PR, what kind of issue are we discussing here ?

Reply:
Isn't the milestone for PRs optional? The milestone label is required to create release notes.

Reply:
How can it be optional if we have a job to enforce it ?

Reply:
I really need to send the governance PR so that we have this documented. As we discussed and agreed on in one of maintainers' meetings:
- issues must be added to the milestone
- PRs must have the milestone label
- PRs with no corresponding issues are added to the milestone for tracking

The original intent was to add the check for the milestone label, but we ended up with this milestone check. Let's agree on the requirement and then we can adjust CI jobs.

Reply:
Agreed, we need a doc to implement what we decided.

Reply:
https://github.com/kyverno/website/pull/1261